Amdad_home.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Amdad Homepage</title>
+    <link rel="stylesheet" href="home_style.css" >
+</head>
+<body>
+    <!-- Button group for navigation though website-->
+    <div class="btn-group">
+       <a href="Amdad_home.html">
+           <button>Home</button>
+       </a>
+       <a href="About_amdad.html">
+           <button>About Amdad</button>
+       </a>
+       <a href="Amdad_form.html">
+           <button>Your thoughts?</button>
+       </a>
+    </div>
+    <div class="rest">
+        
+        <br>
+        <br>
+        <br>
+        <h1 class="title">
+            Meet Amdadur Rahman.
+            <br>
+            <br>
+            Coolest Guy, 
+            <br>
+            Beautiful Being.
+        </h1>
+   
+        
+    </div>
+        
+    <div class="container">
+        <img src="Asfand.JPG" alt="Amdad" style="width:40%" class="image">
```

Comment:
Alt tags should act like captions, so if my internet was slow or I was using a screen reader I would know exactly was missing. The browser will show us it is a broken image, or screen reader will state its an image from the HTML tag so avoid stating its an image and rather use a caption like "Amdad headshot"